provision/ubuntu/install.sh

```diff
@@ -3,7 +3,8 @@
 set -eu
 
 source "${ENV_FILEPATH}"
-export 'IPROUTE_BRANCH'=${IPROUTE_BRANCH:-"v4.14.0"}
+export 'IPROUTE_BRANCH'=${IPROUTE_BRANCH:-"4.20.0-1ubuntu0bjn2"}
+export 'IPROUTE_GIT'=${IPROUTE_GIT:-https://github.com/joestringer/iproute2}
```

Comment:
Nit: could we add a small comment saying _"change to git.kernel.org/... after the following patches has been accepted: $URL1, $URL2, ..."_?

Reply:
This kind of change will need to be driven from the Cilium side, as the BPF and ELF handling code is relying on the implementation in these patches. There's no specific patches proposed for this right now, and lots of potential paths to fix this. I've filed an issue below with the details, depending on which path we take it will require changing this, as a result I don't see a lot of value in adding a code comment here in this repo.

https://github.com/cilium/cilium/issues/7123